neovim/plugin/script_host.py

```diff
@@ -227,6 +229,13 @@ def find_module(fullname, path=None):
         @staticmethod
         def find_spec(fullname, path=None, target=None):
             """Method for Python 3.4+."""
+            # Check sys.modules, required for importlib.reload.
+            # path appears to be the module then, and calling find_spec with it
+            # fails.
+            try:
+                return sys.modules[fullname].__spec__
```

Comment:
`__spec__` might be None here.. seen when using jedi-vim, which apparently also comes here.

Need to investigate..

Reply:
@davidhalter
Any idea maybe?
Maybe AttributeError could be handled like KeyError, but it would be good to know what is going on.

I do not have the details anymore, but am using this branch again now (to fix `reload(jedi_vim)` in Neovim).